joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h

```diff
@@ -277,6 +286,13 @@ class HardwareInterfaceAdapter<hardware_interface::EffortJointInterface, State>
       }
     }
 
+    // Load velocity feedforward gains from parameter server
```

Comment:
Can this be reused somehow?
It's the same as in line 185..
FF-enabled base class?

Reply:
In fact, most code is duplicated between the `VelocityJointInterface` and the `EffortJointInterface` adapters. I'll refactor into a common base class for these specializations.